esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -206,9 +208,9 @@ struct WUComponentLogOptions
             logDataFormat = logAccessFormatFromName(requestedLogDataFormat.str());
 
         StringBuffer start; // Absolute query time range start in YYYY-DD-MMTHH:MM:SS
-        zapHttpRequest->getParameter("LogFilter_AbsoluteTimeRange_Start", start);
+        zapHttpRequest->getParameter("LogFilter_AbsoluteTimeRange_StartDate", start);
```

Comment:
@jeclrsg I noticed the zap request absolute time range hasn't been working, and noticed this discrepancy in the param name. Can you confirm this is the correct name? This is targeting 9.2.x and it needs to be in synch w/ eclwatch for all versions.

Reply:
Yes, LogFilter_AbsoluteTimeRange_StartDate and LogFilter_AbsoluteTimeRange_EndDate are correct

Reply:
Thanks @jeclrsg has that been the case since 9.2.x?

Reply:
That's right, everything 9.2.x and up